detox/test/e2e/12.animations.test.js

```diff
@@ -51,7 +51,13 @@ describe('React-Native Animations', () => {
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).not.toExist();
     });
 
-    it(`should wait during delays shorter than 1.5s`, async () => {
+    // todo: investigate test failure on new-arch.
```

Comment:
Is this to-do still valid?

Reply:
Definitely, forgot to open the issue for that, + I'll add a comment in the release notes as well. JS animations behave quite different now and needs to be sync'd differently

Reply:
https://github.com/wix/Detox/issues/4706

Reply:
👀